generictry/src/main/java/generictry/agregate1/A.java

```diff
@@ -0,0 +1,34 @@
+package generictry.agregate1;
+
+// TODO: Auto-generated Javadoc
+/**
+ * The Class A.
+ *
+ * @author Jean-Aymeric Diet
+ * @param <TB>
+ *          the generic type
+ */
+public class A<TB extends B> {
+
+	/** The b. */
+	private TB b;
+
+	/**
+	 * Gets the b.
+	 *
+	 * @return the b
+	 */
+	public TB getB() {

```

Comment:
\[Checkstyle\] ERROR: Method 'getB' is not designed for extension \- needs to be abstract, final or empty\.